src/k8s/pkg/component/component_test.go

```diff
@@ -102,6 +103,26 @@ func mustMakeMeSomeReleases(store *storage.Storage, t *testing.T) (all []*releas
 	return all
 }
 
+func mustMakeMeSomeComponents() map[string]types.Component {
```

Comment:
a minor comment: the `must` convention in Go indicates that you perform some setup steps that could fail and then panic.
This function should can just be a variable declaration. You don't perform any setup steps.